debezium-core/src/main/java/io/debezium/config/Field.java

```diff
@@ -1175,4 +1175,19 @@ public static int isDouble(Configuration config, Field field, ValidationOutput p
         return 0;
     }
 
-}
\ No newline at end of file
+    /**
+     * Validates the field only for being a not-null value.
+     *
+     * The validation here is more relaxed than in {@link Field::isRequired}. In contrast to {@link Field::isRequired},
+     * this method does not check if the string has more than zero characters.
+     */
+    public static int isNotNull(Configuration config, Field field, Field.ValidationOutput problems) {
```

Comment:
Thanks for the JavaDoc. This PR no longer needs this method, so should we keep it?